openlibrary/templates/search/sort_options.html

```diff
@@ -1,9 +1,31 @@
 $def with (selected_sort, exclude=None, default_sort='relevance', search_scheme='works')
 
-<span class="tools">
-    <img src="/images/icons/icon_sort.png" class="" alt="$_('Sorting by')" width="9" height="11" class="sorter" />
-    $_('Sorted by'):
+<span class="sort">
 $code:
+     sort_option_names = {
+      'relevance': _("Relevance"),
+      'work_count desc': _("Work Count"),
```

Comment:
What does work_count sort by?

Reply:
I believe, it would have sorted by the number of works an author had written.